frontend/src/components/App/CreateCluster/AddCluster.tsx

```diff
@@ -81,9 +82,17 @@ export default function AddCluster(props: DialogProps & { onChoice: () => void }
               </CardContent>
             </Card>
           </Grid>
-          {addClusterProviders.length > 0 && (
+          <Grid item xs={12}>
+            <Typography variant="h4">{t('translation|Providers')}</Typography>
+          </Grid>
+          {isElectron() && addClusterProviders.length === 0 && (
             <Grid item xs={12}>
-              <Typography variant="h4">{t('translation|Providers')}</Typography>
+              <Button
+                onClick={() => history.push('/#/plugin-catalog/headlamp-plugins/headlamp_minikube')}
+                startIcon={<InlineIcon icon="mdi:plus-box-outline" />}
+              >
+                {t('translation|Add Local Cluster Provider')}
+              </Button>
```

Comment:
For this to be compatible with builds that may not use the plugin catalog, we should probably check if the plugin catalog plugin is available and enabled.

Reply:
Is there one you know of?

Reply:
Added a check for plugin catalog.